src/main/java/com/book/controller/UserController.java

```diff
@@ -0,0 +1,48 @@
+package com.book.controller;
+
+import com.book.DTO.UserDTO;
+import com.book.model.JwtModel;
+import com.book.model.UsersEntity;
+import com.book.service.MyUserDetailsService;
+import com.book.service.UserService;
+import com.book.util.JwtUtils;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@CrossOrigin(origins = "http://localhost:3000")
+@RestController
+@RequestMapping("/user")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+    private final JwtUtils jwtUtils;
+
+    @PostMapping ("/changePassword")
+    public ResponseEntity<?> changePassword(@RequestBody UserDTO userDTO) {
+        UsersEntity usersEntity = new UsersEntity();
+        usersEntity.setPassword(userDTO.getPassword());
+        usersEntity.setUserName(jwtUtils.getUserNameFromJwtToken(userDTO.getJwt()));
+
+        if (userService.updateUser(usersEntity).isEmpty()) {
+            return new ResponseEntity<>("User failed to update", HttpStatus.BAD_REQUEST);
+        }
+        return new ResponseEntity<>("User updated successfully", HttpStatus.OK);
+    }
+
+    @PostMapping ("/getUser")
+    public ResponseEntity<?> getUser(@RequestBody ObjectNode json) {
```

Comment:
also, update this feature to use DTO param.
And it should return `ResponseEntity< UsersEntity>`, not `ResponseEntity<?>`